about/about.html

```diff
@@ -0,0 +1,126 @@
+
+    <section class="intro-section intro-section-about padding-x text-center">
+      <article class="content content-about">
+        <header class="title">
+          <h1 class="display-1 display-1-about text-secondary">18th Latin American Congress on Free Software and Open
+            Technologies</h1>
+        </header>
+        <p class="par">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Harum aperiam sunt ut esse delectus,
+          aliquam ea reiciendis laudantium maxime aliquid. Sed aliquam harum obcaecati consectetur consequatur soluta
+          nihil eveniet blanditiis, sunt repudiandae excepturi delectus corporis illum natus placeat dignissimos
+          aspernatur? Ex dignissimos nisi aliquam laudantium? Temporibus consequatur tempora laboriosam possimus.
+        </p>
+        <div class="intro-bottom">
+          <p class="par">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Autem cumque molestias voluptate
+            blanditiis placeat rem?</p>
+          <a class="text-primary" href="#">Hello!123@fake.com</a>
+        </div>
+      </article>
+    </section>
+    <!-- ************************************************************************************************************************
+                                                    LOGO SECTION
+    ************************************************************************************************************************ -->
+    <section class="logo-section padding-y padding-x flex text-center">
+      <h1 class="display-3 text-primary">Latin American Congress on Free Software and Open Technologies's Logo</h1>
+      <div class="section-line"></div>
+      <p class="par par-logo">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Nam esse laudantium nulla porro eius
+        optio accusantium? A atque harum itaque.</p>
+      <div class="logo logo-about"></div>
+    </section>
+    <!-- ************************************************************************************************************************
+                                                    SPEAKERS SECTION
+    ************************************************************************************************************************ -->
+    <section class="speakers-section speakers-section-about padding-y padding-x text-center flex">
+      <h1 class="h3 text-primary">Featured Speakers</h1>
+      <div class="section-line"></div>
+      <p class="par"> Lorem ipsum dolor sit amet consectetur, adipisicing elit. Impedit, illo! Possimus, placeat.</p>
+      <article class="image-container flex">
+        <figure class="person-img-wrapper">
+          <img class="congress-img" src="../assets/images/main/congress-1.png" alt="congress-1">
+        </figure>
+        <figure class="person-img-wrapper">
+          <img class="congress-img" src="../assets/images/main/congress-2.png" alt="congress-2">
+        </figure>
+      </article>
+    </section>
```

Comment:
- [ ] This section looks a bit different on the desktop version from the design guidelines in Behance. Please consider following the required design by making the images to be aligned horizontally on the desktop version of the app.

- [ ] Also, please consider removing the Featured Speakers here as there is no featured speaker section on the about page and consider adding the correct details.

Design guideline on desktop for the about page:

<img width="557" alt="Loom_Ys1MIbs72e" src="https://user-images.githubusercontent.com/81468675/147155203-432f4df3-bc95-4ade-9859-7e4b48a1b573.png">

Yours:
<img width="415" alt="Loom_vAguhp1B0M" src="https://user-images.githubusercontent.com/81468675/147155250-c0d2d90f-f880-4efe-92fa-787583309c8e.png">